backend/src/routes/api/nb-events/eventUtils.ts

```diff
@@ -0,0 +1,95 @@
+import { V1EventList } from '@kubernetes/client-node';
+import { KubeFastifyInstance, NotebookStatus } from '../../../types';
+
+export const getNotebookEvents = async (
+  fastify: KubeFastifyInstance,
+  nbName: string,
+): Promise<NotebookStatus> => {
+  const response = await fastify.kube.coreV1Api
+    .listNamespacedEvent(
+      fastify.kube.namespace,
+      undefined,
+      undefined,
+      undefined,
+      `involvedObject.kind=Notebook,involvedObject.name=${nbName}`,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      false,
+    )
+    .then((res) => {
+      return res.body as V1EventList;
+    })
+    .catch((e) => {
+      fastify.log.error(e);
+      return { items: [] } as V1EventList;
+    });
+  let percentile = 0;
+  let status: 'In Progress' | 'Warning' | 'Error' | 'Success' = 'In Progress';
```

Comment:
```suggestion
  let status: EventStatus = EventStatus.IN_PROGRESS;
```
Please make an enum so we don't have magic strings.

Reply:
Added enum to backend types.